.github/workflows/continuous-integration.yml

```diff
@@ -116,3 +102,24 @@ jobs:
         run: npm run build
       - name: Verify doc changes
         run: ./bin/verify-doc-changes.sh
+
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name.schedule }}
+    needs: [ test, lint ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+      - name: Bootstrap project
+        run: |
+          npm ci --ignore-scripts
+          npm run postinstall
+      - name: Build project
+        run: npm run build
+      - name: Publish packages
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: |
+          ./node_modules/.bin/lerna version
+          ./node_modules/.bin/lerna publish from-git --yes
```

Comment:
We may want to put a note in the [Releasing New Versions docs](https://github.com/strongloop/loopback-next/blob/9f754ce5fd3a6e6849e7b7a2cbe647f6701955ea/docs/site/DEVELOPING.md#releasing-new-versions) that states new releases are published automatically by this CI script.

Reply:
Let's use `npx lerna` instead.

Reply:
@raymondfeng I don't think it's a good idea to use npx. If there is a major and something changes it will break.

@achrinza It is necessary? They can also be generated manually

Reply:
npx uses the locally installed lerna before it tries to download.

Reply:
@raymondfeng Thanks for the clarification, I didn't know

Reply:
Done

Reply:
@achrinza What do you think of the changes in DEVELOPING.md?

Reply:
LGTM :+1: